rts/Lua/LuaSyncedRead.cpp

```diff
@@ -1280,6 +1281,31 @@ int LuaSyncedRead::GetMapOptions(lua_State* L)
 }
 
 
+/***
+ *
+ * @function Spring.GetModOption
+ *
+ * @string modOption 
+ *
+ * @treturn string value of modOption in option map
+ */
+int LuaSyncedRead::GetModOption(lua_State* L)
+{
+	const auto& modOpts = CGameSetup::GetModOptions();
+	
+	const std::string& opt = luaL_checkstring(L, 1);
+
+	if (modOpts.find(opt) == modOpts.end()) {
+		return 0;
+	}
+	else {
+		const std::string* optValue = modOpts.try_get(opt);
+		lua_pushsstring(L, *optValue);
+		return 1;
+	}
+}
```

Comment:
Isn't the `find` redundant with `try_get` below? How about
```suggestion
	const std::string* optValue = modOpts.try_get(opt);
	if (optValue == nullptr)
		return 0;

	lua_pushsstring(L, *optValue);
	return 1;
}
```